administrator/components/com_guidedtour/src/Controller/DisplayController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace JatinSalve\Component\Guidedtour\Administrator\Controller;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\MVC\Controller\BaseController;
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_helloworld
```

Comment:
Try not to use some example components, they are mostly outdated. Look into the Joomla! core and take infors/data from there.